homeassistant/components/sensibo/switch.py

```diff
@@ -66,6 +79,16 @@ def build_params(command: str, device_data: SensiboDevice) -> dict[str, Any] | N
             "acState": {**device_data.ac_states, "on": new_state},
         }
         return params
+    if command == "set_pure_boost":
```

Comment:
We should clean this up in the future per the comment here https://github.com/home-assistant/core/pull/73687#discussion_r902363239

But for a future PR as we want to get this one merged ASAP to avoid the breaking change

Reply:
Yes. I just wanted to get this one in (as per prior discussions) and then do a bigger clean-up in a new PR as there is more things.
This grew a a bit along the way 😄
I will hold off new features until clean-up done.